src/envdrift/scanner/git_hound.py

```
@@ -0,0 +1,748 @@
+                                try:
+                                    findings_data.append(json.loads(line))
+                                except json.JSONDecodeError:
+                                    continue
+
+                        for item in findings_data:
+                            finding = self._parse_finding(item, path)
+                            if finding:
+                                all_findings.append(finding)
+                                total_files += 1
+                    except Exception:
+                        pass  # Parsing failed, continue
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)